### PR TITLE
systelab-numpad: Use a Two-Way Data Binding property for the value

### LINKED ISSUE #44: systelab-numpad: Use a Two-Way Data Binding property for the value.

### PR DESCRIPTION
**Pull request by olgapuig**
Fix #44

### CODE REVIEW

File: src/app/showcase/components/input/showcase-input.component.html [hunk 1]
```diff
@@ -36,7 +36,7 @@
     <div class="row mt-1">
         <label class="col-md-3 col-form-label">NumPad</label>
         <div class="col-md-9">
-            <systelab-numpad></systelab-numpad>
+            <systelab-numpad [(numpadValue)]="value" (numpadValueEnter)="doEnter()"></systelab-numpad>
```
Comment (alfonsserra) [inline src/app/showcase/components/input/showcase-input.component.html]: Use the more standard [(value)] and (change)

File: src/app/systelab-components/numpad/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Component to show a Numeric Keyboard dialog for an Input Text.
 ## Using the component
 
 ```html
-<systelab-numpad [numpadValue]="value" (numpadValueEmitter)="doSelectValue($event)"></systelab-numpad>
+<systelab-numpad [numpadValue]="value" (numpadValueEnter)="doSelectValue($event)"></systelab-numpad>
```
Comment (alfonsserra) [inline src/app/systelab-components/numpad/README.md]: Use the more standard [(value)] and (change)

File: src/app/showcase/components/input/showcase-input.component.html (hunk 1 above)
Comment (olgapuig) [reply src/app/showcase/components/input/showcase-input.component.html]: Done

File: src/app/systelab-components/numpad/README.md (hunk 2 above)
Comment (olgapuig) [reply src/app/systelab-components/numpad/README.md]: Done

**Pull request merged**